libs/blocks/global-navigation/utilities/getUserEntitlements.js

```diff
@@ -54,7 +124,10 @@ const getSubscriptions = async ({ queryParams, locale }) => {
         'Accept-Language': getAcceptLanguage(locale).join(','),
       },
     })
-    .then((response) => (response.status === 200 ? response.json() : null));
+    .then((response) => {
+      clearTimeout(timeout);
+      return response.status === 200 ? response.json() : null;
```

Comment:
if the status is not 200, will `null` bill replaced with the `emptyEntitlements`?

Reply:
Nope, great catch - fixed.